conformance/utils/suite/experimental_suite.go

```diff
@@ -282,3 +290,50 @@ func (suite *ExperimentalConformanceTestSuite) Report() (*confv1a1.ConformanceRe
 		ProfileReports:    profileReports.list(),
 	}, nil
 }
+
+// ParseImplementation parses implementation-specific flag arguments and
+// creates a *confv1a1.Implementation.
+func ParseImplementation(org, project, url, version, contact string) (*confv1a1.Implementation, error) {
+	if org == "" {
+		return nil, errors.New("implementation's organization can not be empty")
+	}
+	if project == "" {
+		return nil, errors.New("implementation's project can not be empty")
+	}
+	if url == "" {
+		return nil, errors.New("implementation's url can not be empty")
+	}
+	if version == "" {
+		return nil, errors.New("implementation's version can not be empty")
+	}
+	contacts := strings.SplitN(contact, ",", -1)
+	if len(contacts) == 0 {
+		return nil, errors.New("implementation's contact can not be empty")
+	}
+
+	// TODO: we may want to add validation for url
+	// TODO: we may want to add validation for version
+	// TODO: we may want to add validation for contact (e.g., @someone or someone@acme.org)
```

Comment:
These would be a good thing to capture as follow up issues, and then link those issues here so they don't end up getting out of band.

Reply:
https://github.com/kubernetes-sigs/gateway-api/issues/2178